egs/wsj/s5/utils/data/get_utt2dur.sh

```diff
@@ -68,7 +68,14 @@ else
       echo  "$0: wav-to-duration is not on your path"
       exit 1;
     fi
-    if ! wav-to-duration scp:$data/wav.scp ark,t:$data/utt2dur 2>&1 | grep -v 'nonzero return status'; then
+
+    read_entire_file=false
+    if [ `cat $data/wav.scp | sed -n '/sox.*speed/p' | wc -l` -gt 0 ]; then
```

Comment:
I think a grep or grep -E command with output piped to /dev/null would be more elegant-- grep returns true if it matched at least one line.